modules/ztp-acm-installing-disconnected-rhacm.adoc

```diff
@@ -16,6 +16,7 @@ You use {rh-rhacm-first} on a hub cluster in the disconnected environment to man
 [NOTE]
 ====
 If you want to deploy Operators to the spoke clusters, you must also add them to this registry.
+See link:https://docs.openshift.com/container-platform/4.8/operators/admin/olm-restricted-networks.html#olm-mirror-catalog_olm-restricted-networks[Mirroring an Operator catalog]
```

Comment:
"See link:https://docs.openshift.com/container-platform/4.8/operators/admin/olm-restricted-networks.html#olm-mirror-catalog_olm-restricted-networks[Mirroring an Operator catalog]" feels incomplete. Would you mind adding a bit:
"See link:https://docs.openshift.com/container-platform/4.8/operators/admin/olm-restricted-networks.html#olm-mirror-catalog_olm-restricted-networks[Mirroring an Operator catalog] for more information."?

Reply:
Changed.